src/store/Network.ts

```diff
@@ -311,10 +311,25 @@ export default {
                     progressTotalNumOfNodes: 1,
                 });
                 nodeNetworkModelResult = await networkService.getNetworkModel(newCandidateUrl, networkType, isOffline).toPromise();
+                let wsUrl = undefined;
+                let wsConnection = undefined;
+                if (navigator.onLine && !isOffline) {
+                    wsUrl = nodeNetworkModelResult.repositoryFactory.createListener().url;
+                    wsConnection = await networkService.checkWebsocketConnection(wsUrl);
+                    if (!wsConnection.status && wsConnection.sslNode) {
+                        nodeNetworkModelResult = await networkService
+                            .getNetworkModel(newCandidateUrl.replace('https', 'http').replace('3001', '3000'), networkType, isOffline)
```

Comment:
I think ports should not be hardcoded and returned by statistics-service

Reply:
this is for the fallback case, sometimes the node allows WS connection on  port 3000 but not 3001, while the node itself is providing ssl connection, so we need to replace the returned one and retry.